source/blueprints/banking.rst

```diff
@@ -11,12 +11,12 @@ In this step-by-step walkthrough, you'll build a conversational application that
 1. The Use Case
 ^^^^^^^^^^^^^^^
 
-In a hands-free manner, from the convenience of their homes, users should have the same food-ordering experience as when conversing with a restaurant waiter or a drive-through attendant. The food is ordered for delivery from a service similar to Amazon Restaurants, Grubhub or DoorDash.
+Users should be able to securely access their banking information and complete tasks as if you are using a mobile banking application or speaking to a teller with natural language. They should be able to check balances, pay off credit card debt, transfer money, and carry out various other banking operations they may perform on day to day basis. 
 
```

Comment:
Users should be able to securely access their banking information and complete tasks as if they are conversing with a teller. They should be able to check balances, pay off credit card debt, transfer money, and carry out various other banking operations they may perform on day to day basis.

Reply:
Add a description of how we expect a front end application to handle secure authentication and pass a user token to the MindMeld server via the request context (which is immutable). The MindMeld server can then use this token to retrieve user information from a secure database. In this blueprint we demonstrate how this can be done by simply passing the user_name in the context and using it to retrieve information from a simple database. For a production app, these can be replaced by a secure token and an API call to a secure database.

Reply:
Actually as noted below, more of the details of this description can go in the KB section, but might be worth briefly mentioning here at a high level. 

In addition where you first start giving example interaction, give an example of how to pass the context, and include a table of the usernames that people can use.